river/tree/extremely_fast_decision_tree.py

```diff
@@ -229,17 +279,22 @@ def _process_nodes(self, x, y, sample_weight, node, parent, branch_index):
 
             if (new_weight - old_weight) >= self.min_samples_reevaluate:
                 # Reevaluate the best split
-                stop_flag = self._reevaluate_best_split(node, parent, branch_index)
+                stop_flag = self._reevaluate_best_split(
+                    node,
+                    parent,
+                    branch_index,
+                    splitter=self.splitter,
```

Comment:
I added a comment to clarify their purpose. They could be passed as parameters, rather than kwargs. However, future generations might want to create variations of EFDT, so this gives us flexibility.